### PR TITLE
Relax Colors.jl to 0.9–0.12

### PR DESCRIPTION
**Pull request by mossr**
For compatibility with Flux 0.8, used by POMDPStressTesting.

### CODE REVIEW

**Comment by MaximeBouton** [comment]
Thank you! 
(will merge once the test are passing)

**Review by MaximeBouton — APPROVED** [review]

**Comment by mossr** [comment]
Awesome, thank you! Looks like they timed out and are now restarting.

Once this is merged, could you bump to v0.1.1 so I can [compat] with this version? Thanks again.

**Comment by MaximeBouton** [comment]
will do, I restarted the build on macos as it was failing for an unknown reason

**Pull request merged**
